Code/EnginePlugins/RmlUiPlugin/Components/RmlUiMessages.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <RmlUiPlugin/RmlUiPluginDLL.h>
+
+struct EZ_RMLUIPLUGIN_DLL ezMsgReload : public ezMessage
```

Comment:
This is a very generic message name, can we give this a more specific name ? e.g. add "Rml"